javascript-typing-tutor/script.js

```diff
@@ -21,24 +24,38 @@ document.addEventListener('keydown', function (e) {
       $background.className = 'background-color-off';
       $opacity.className = 'opacity text';
       $modalText.textContent = Math.round(value / attempts * 100) + '% with ' + (attempts - value) + ' errors.';
-      $retry2.addEventListener('click', function () {
-        if (test === true) {
-          location.reload();
-        }
-      });
     }
   } else {
     $text[value].className = 'wrong-text under-line';
     attempts++;
   }
+}
+
+function modalRetry(e) {
+  if (e) {
+    value = 0;
+    attempts = 0;
+    $modal.className = 'modal hidden';
+    $background.className = '';
+    $opacity.className = 'text';
```

Comment:
This can probably just be done for both "retry" buttons, that way you only need one named function for resetting the app.